src/app/core/admin/admin-entity/admin-entity-edit/admin-entity-edit.component.html

```diff
@@ -0,0 +1,77 @@
+<h2 i18n>Configure Static Field "{{ entityConstructor.label }}"</h2>
+<p i18n>
+  The entity settings here apply to the record type overall and affect both the
+  entity here in the current view as well as all other forms and lists where
+  this entity is displayed.
+</p>
+
+<form [formGroup]="staticDetailsform">
+  <mat-tab-group formGroupName="staticDetails">
+    <mat-tab label="Basics" i18n-label>
+      <div class="grid-layout flex-grow margin-top-regular">
+        <div class="entity-form-cell">
+          <mat-form-field>
+            <mat-label i18n>Label</mat-label>
+            <input formControlName="label" matInput #formLabel />
+          </mat-form-field>
+
+          <mat-form-field floatLabel="always">
+            <mat-label i18n>
+              Label (Plural)
+              <fa-icon
+                icon="question-circle"
+                matTooltip="Optionally you can define an additional Plural label to be displayed in Entity header."
+                i18n-matTooltip
+              ></fa-icon>
+            </mat-label>
+            <input
+              formControlName="labelPlural"
+              matInput
+              [placeholder]="formLabel.value"
+            />
+          </mat-form-field>
+
+          <mat-form-field floatLabel="always">
+            <mat-label i18n>
+              Icon
+              <fa-icon
+                icon="question-circle"
+                matTooltip="Optionally you can Change the Icon of Entity types in navbar."
+                i18n-matTooltip
+              ></fa-icon>
+            </mat-label>
+            <input
+              formControlName="icon"
+              matInput
+              [placeholder]="formLabel.value"
+            />
```

Comment:
Check out whether we can use a component to visually select fontawesome icons.
Maybe the following or any other that you might find: https://www.npmjs.com/package/ngx-icon-picker

Reply:
Thanks for the suggestion. However, I've looked into it, and unfortunately, `ngx-icon-picker `doesn't support Angular 17, and I'm currently unable to find another library that does.